### PR TITLE
Add guide for upgrading to v2

### PR DESCRIPTION
**Pull request by rmccue**
@roborourke @joehoyle Anything I've missed here?

### CODE REVIEW

File: other-docs/guides/upgrading/v2.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+If you have an existing Local Chassis install, a manual step is required to update. Run the following from your project directory:
+
+```sh
+cd chassis/extensions/platform-chassis-extension
+git pull
+vagrant provision
+```
+
+Additionally, new convenience commands are available, including `composer chassis shell`. See the [command documentation](docs://local-chassis/#available-commands) for full information.
```
Comment (joehoyle) [inline other-docs/guides/upgrading/v2.md]: We might want to mentioned that Chassis doesn't support Analytics / Experiment results yet.

File: other-docs/guides/upgrading/v2.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Upgrading to v2
+
+## Breaking Changes
+
+There are no known breaking changes in version 2.
+
+
```
Comment (joehoyle) [inline other-docs/guides/upgrading/v2.md]: Is it worth adding an "update `^1.0.0`, to `2.0.0` and deploy to upgrade to Altis version 2."

File: other-docs/guides/upgrading/v2.md (hunk 1 above)
Comment (roborourke) [reply other-docs/guides/upgrading/v2.md]: I should probably do that, think it's easy enough to add docker stuff via puppet.

**Comment by roborourke** [comment]
Can't think of anything major missed out

File: other-docs/guides/upgrading/v2.md (hunk 2 above)
Comment (rmccue) [reply other-docs/guides/upgrading/v2.md]: Yeah, although that's already covered under the main upgrading guide so didn't think it was worth repeating here.

**Comment by joehoyle** [comment]
just remembered an upgrade bug I’ve seen a  few times. Because we added new plugins to the whitelist in the core installer, the first “update” is running the older composer plugin, so the new plugins get installed to content/plugins. It’s not until you do a another update or a fresh install that the plugins will go to the right place.

If we aren’t going to fix that in beta 2, we should document this

**Pull request merged**